tasks/main.yml

```diff
@@ -84,7 +84,7 @@
 
 - name: deploy SSH daemon configuration
   template:
-    src: 'sshd_config_{{ ansible_os_family | lower }}.conf'
+    src: 'sshd_config_{{ ansible_distribution | lower }}_{{ ansible_distribution_major_version | lower }}.conf'
     dest: /etc/ssh/sshd_config
     owner: root
     group: root
```

Comment:
Can we do this with `with_first_found`? Something like:

```yml
- name: deploy SSH daemon configuration
  template:
    src: '{{ item }}'
    ...
  with_first_found:
    - files:
        - 'sshd_config_{{ ansible_distribution | lower }}_{{ ansible_distribution_major_version }}.conf'
        - 'sshd_config_{{ ansible_os_family | lower }}.conf'
```

This would simplify things for RHEL/CentOS and Arch.

Reply:
amended commit and resolved the issue